packages/2bwm/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/venam/2bwm"
+TERMUX_PKG_DESCRIPTION="A fast floating WM written over the XCB library and derived from mcwm."
+TERMUX_PKG_LICENSE="Unlicense"
```

Comment:
The license of `2bwm` is not `Unlicense` but `ISC`. See the head of `2bwm.c` for example. The source tarball does not have a separate license file, so you need to extract the license notice from the source and put it in a separate file.